### ISSUE TITLE
"Invalid attributes set to undefined"

### ISSUE THREAD

**Issue by kevinburkeshyp** [balderdashy/waterline#1118]
If the caller does not set the `model` attribute on a error dictionary, Waterline returns an error that says "Invalid attributes sent to undefined". The sails-postgresql library at least does not set this field. 

I believe this would be customer facing if you used this in a blueprint route, but we're in the process of ripping those out of our codebase, so I haven't had time to investigate.

**Comment by sailsbot** [comment]
Thanks for posting, @kevinburkeshyp.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
